phpstan.neon

```diff
@@ -46,36 +41,6 @@ parameters:
     ignoreErrors:
         -
             identifier: missingType.generics
-        -
-            message: '#Class cognitive complexity is \d+, keep it under \d+#'
-            paths:
-                - rules/Php70/EregToPcreTransformer.php
-                - src/NodeTypeResolver/NodeTypeResolver.php
-                - src/NodeTypeResolver/PHPStan/Scope/PHPStanNodeScopeResolver.php
-                - src/PHPStanStaticTypeMapper/TypeMapper/UnionTypeMapper.php
-                - src/PhpParser/Printer/BetterStandardPrinter.php #41
-                - src/PhpDocParser/PhpDocParser/PhpDocNodeTraverser.php
-                - rules/TypeDeclaration/Rector/ClassMethod/StrictArrayParamDimFetchRector.php
-                - src/CustomRules/SimpleNodeDumper.php
-
-        -
-            message: '#Cognitive complexity for "(.*?)" is (.*?), keep it under 11#'
-            paths:
-                - rules/Php70/EregToPcreTransformer.php
-                - src/PhpParser/Node/Value/ValueResolver.php
-                - rules/DeadCode/NodeManipulator/LivingCodeManipulator.php
-                - rules/Php80/NodeResolver/SwitchExprsResolver.php
-                - src/NodeTypeResolver/NodeTypeResolver.php
-                - src/BetterPhpDocParser/PhpDocParser/DoctrineAnnotationDecorator.php
-                - src/Parallel/Application/ParallelFileProcessor.php
-                # rector rules
-                - rules/*/Rector/*Rector.php
-                - utils/Rector/MoveAbstractRectorToChildrenRector.php
-                - src/PhpDocParser/PhpDocParser/PhpDocNodeTraverser.php
-                - src/NodeTypeResolver/PHPStan/Scope/PHPStanNodeScopeResolver.php
-                - src/Configuration/RectorConfigBuilder.php
-                - rules/TypeDeclaration/PHPStan/ObjectTypeSpecifier.php
-                - src/PhpAttribute/NodeFactory/PhpNestedAttributeGroupFactory.php
```

Comment:
While cognitive complexity package might bring some value, these tedious ignores that spam our phpstan.neon makes it rather discouraing. I have to admit it doesn't help us here much :D lets get rid of it and code more freely